src/main/java/org/commonwl/view/workflow/Workflow.java

```diff
@@ -353,6 +356,16 @@ public void setLicenseLink(String licenseLink) {
     this.licenseLink = licenseLink;
   }
 
+  public String getLicenseName() {
+    if (licenseLink == null) {
+      return null;
+    }
+    if (licenseLink.startsWith(LicenseUtils.SPDX_PREFIX)) {
+      return SpdxLicense.fromId(licenseLink.replace(LicenseUtils.SPDX_PREFIX, "")).name;
+    }
+    return licenseLink;
```

Comment:
This looks a lot like the code in `CWLService`. Do you think there would be some way to reuse this common part? (Assuming that would make sense and prevent these blocks from getting out of sync)

Reply:
This code does the opposite operation w.r.t. the one in `CWLService`, going from URL to common name. THe purpose is to give users a better UX by visualizing the name (e.g., "Apache Licende 2.0") instead of the raw link (e.g., "https://spdx.org/licenses/Apache-2.0")

Reply:
My bad, I didn't see that in one case it returned the license link, and in the other one it did something else :+1: